apps/showcase/src/assets/trainings/sdk/steps/model-extension/exercise/core/flight/flight.ts

```diff
@@ -0,0 +1,9 @@
+/* TODO create the type FlightCoreIfy which extends Flight, imported from the ../base folder */
+/*   Add an extra field `id: string` */
```

Comment:
I don't know if you need this line since you mentioned it in the instructions

Reply:
Should we also mention in the instructions the need of `IgnoreEnum` ? Or ask the user to check out the documentation to be used as a template ?

Reply:
Even if this is already mentioned in the instructions, I think it's a good thing to put the instructions also directly in the file.

As for `IgnoreEnum`, I prefer to keep the exercise as simple as possible. No need to mention it here as it is not doing anything in this simple case.